00a-Setup/Readme.md

```diff
@@ -5,7 +5,6 @@
  <!-- **Release Version: AzSDK 2.11.xx <=> AzSK 3.0.x ** -->
 >**Pre-requisites**:
 > - PowerShell 5.0 or higher. 
-> - AzureRM Version 5.2.0
```

Comment:
Shouldn't we update the new dependency here?

Reply:
Since we have removed AzureRm module dependency, this has been removed. Added "Run command 'Find-Module AzSK -includedependencies' to see all dependencies." comment as an alternative.

Reply:
Sure, thanks for the clarification.